css/style.css

```diff
@@ -0,0 +1,51 @@
+.graybackground{
+  background-color: rgb(234, 229, 229);
+}
+main{
```

Comment:
Please insert an empty line before the next rule.

Reply:
> lease insert an empty line before the next rule.

- [ ] Kindly explain the why of your comments to make sure the person you are reviewing understand why they're going to make this changes :+1: 

You could say for example 

Please insert an empty line before the next rule, it will make your code more readable :smile: